corehq/apps/translations/app_translations/download.py

```diff
@@ -248,7 +248,8 @@ def get_case_search_rows(langs, module, domain):
 
 def get_module_detail_rows(langs, module):
     rows = []
-    rows += _get_module_detail_no_items_text(langs, module)
+    if hasattr(module, 'case_details'):
```

Comment:
This won't get hit for report modules because `get_module_rows` handles them separately and won't call this function.

(What caught my eye was that the next few lines reference `module.case_details` without checking.)

Reply:
removed - [7699629](https://github.com/dimagi/commcare-hq/pull/32602/commits/76996290cab5f9922946fbe261d6b44eeb88dbce)